wled00/wled.cpp

```diff
@@ -338,7 +338,10 @@ void WLED::setup()
 
   #ifdef WLED_ENABLE_ADALIGHT
   if (!pinManager.isPinAllocated(3)) {
-    // TODO: Should this be reserved for Ada?  Serial input pin?  Else, doesn't appear in pin ownership list....
+    // allow failure for pin allocations (may be used by debug)
+    // TODO: consider if/how DEBUG builds should work with AdaLight enabled
+    pinManager.allocatePin(1, true,  PinOwner::AdaLight);
```

Comment:
Odd, but ADA only needs GPIO3.

Reply:
oops... fixed.   GPIO1 and GPIO3 are now reserved because they are unconditionally used for the serial port.